server/api/collections.js

```diff
@@ -482,26 +478,26 @@ router.post("collections.export_all", auth(), async (ctx) => {
   const team = await Team.findByPk(user.teamId);
   authorize(user, "export", team);
 
-  await Event.create({
-    name: "collections.export",
-    teamId: user.teamId,
-    actorId: user.id,
-    ip: ctx.request.ip,
-  });
-
   if (download) {
```

Comment:
Not sure it needs to be now but the plan will be to remove this flag entirely

Reply:
We are not using this in the app. I'm up for removing this. Is there something stopping us?

Reply:
It might be in use by API consumers, but honestly the whole reason this exercise is happening is to avoid running large exports on the API process so lets remove :)